test/powershell/engine/SemanticVersion.Tests.ps1

```diff
@@ -178,10 +199,6 @@ Describe "SemanticVersion api tests" -Tags 'CI' {
             # Revision isn't supported
             { [SemanticVersion]::new([Version]::new(0, 0, 0, 4)) } | Should Throw # PSArgumentException
             { [SemanticVersion]::new([Version]::new("1.2.3.4")) } | Should Throw # PSArgumentException
-
-            # Build is required
```

Comment:
Under what scenario would we want a default constructor? If you want `0.0.0`, be explicit.